dev/reference/tools/data-migration/releases/1.0.2.md

```diff
@@ -0,0 +1,47 @@
+---
+title: DM 1.0.2 Release Notes
+category: Releases
+---
+
+# DM 1.0.2 Release Notes
+
+Release date: October 30, 2019
+
+DM version: 1.0.2
+
+DM-Ansible version: 1.0.2
+
+## Improvements
+
+- Generate some config items for DM-worker automatically
+- Generate some config items for replication task automatically
+- Simplify the output of query-status without arguments
+- Manage DB connections directly for downstream
+
+## Bug fixes
+
+- Fix some panic when starting up or executing SQL statements
+- Fix abnormal sharding DDL replication on DDL execution timeout
+- Fix starting task failure caused by the checking timeout or any inaccessible DM-worker
+- Fix SQL execution retry for some error
+
+## Detailed Bug Fixes and Changes
+
+- Generate random server-id for DM-worker config automatically [#337](https://github.com/pingcap/dm/pull/337)
```

Comment:
```suggestion
- Generate random `server-id` for DM-worker config automatically [#337](https://github.com/pingcap/dm/pull/337)
```

How about quoting (``` ` ```) the words that have been quoted in https://github.com/pingcap/docs-cn/pull/1995/files.

Reply:
Thanks for reminding me. Updated now.